lib/publishing_api_finder_publisher.rb

```diff
@@ -1,4 +1,5 @@
 require "gds_api/publishing_api_v2"
+require "govuk_schemas"
```

Comment:
You'll also need to move `govuk_schemas` out of the test group in the Gemfile https://github.com/alphagov/search-api/blob/7f9346010101255e6ebd43bbf948a972b68ef1b2/Gemfile#L41-L45